client/modules/static/components/landing/Hero.js.jsx

```diff
@@ -63,7 +63,7 @@ class Hero extends React.Component{
     return (
       <div className="hero main">
         <div className="yt-container">
-          <h1>This is Yote</h1>
+          <h1>This is <div className="nav-otherlogo">Yote</div></h1>
```

Comment:
in genaral DIVs should not be inside text-elements like ```<h1>, <p>, etc```.  this is because DIVs by default are display block.  if you need to put things inside these elements, use other text elements like 
```
<h1> this is <span className="class"> YOTE</span><h1>
```  
or break the H1 into a div class with header-like styling. i.e. 
``` 
<div className="header-one">This is <div className="nav-logo.other">YOTE</div></div>
```